app/assets/javascripts/components/locales/index.jsx

```diff
@@ -22,6 +22,7 @@ import zh_hk from './zh-hk';
 import zh_cn from './zh-cn';
 import bg from './bg';
 import id from './id';
+import kab from './kab';
```

Comment:
Your `kab.jsx` seems missing from this PR. Try adding with `git add kab.jsx` before doing `git commit`.